### PR TITLE
Create functionality for MBStyle max/min brightness on raster layer

### PR DESCRIPTION
**Pull request by johngschulz**
Added brightness support to raster symbolizer

additional testing & update

some refactoring for clarity for RGB bands

cleaning up

attempting to set the correct options for contrast enhancement on each channel(r, g, b)

Clean up

preparing for push

### CODE REVIEW

File: modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java [hunk 1]
```diff
@@ -221,14 +222,44 @@ public Number getFadeDuration() {
...
+//    		    HashMap vendorOptions = new HashMap<>();
+    		    ContrastEnhancement channelCe = sf.contrastEnhancement(ff.literal(1.0), ContrastMethod.NORMALIZE);
+    		    channelCe.addOption("algorithm", ff.literal("ClipToMinimumMaximum"));
+    		    channelCe.addOption("minValue", ff.literal(String.valueOf((double)getBrightnessMin() * 255)));
+    		    channelCe.addOption("maxValue", ff.literal(String.valueOf((double)getBrightnessMax() * 255)));
+//    		    vendorOptions.put("algorithm", "ClipToMinimumMaximum");
+//    		    vendorOptions.put("minValue", String.valueOf((double)getBrightnessMin() * 255)); // 0 -255 are the min/max value for color brightness in geotools  
+//    		    vendorOptions.put("maxValue", String.valueOf((double)getBrightnessMax() * 255)); // brightness scaled to between 0-1 in mapbox
+//    		    channelCe.setOptions(vendorOptions);
+//        }
```
Comment (tbarsballe) [inline modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java]: There is a bunch of commented out code here - it should be removed.

File: modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java [hunk 2]
```diff
@@ -243,7 +274,7 @@ public Number getFadeDuration() {
                 Collections.emptySet(),
                 filter.semanticTypeIdentifiers(),
                 rules));
-    }
+        }
```
Comment (tbarsballe) [inline modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java]: This indentation seems wrong

**Comment by johngschulz** [comment]
I got caught up in QA testing - I've started to look back into it for the fixes, and will update.

**Comment by johngschulz** [comment]
Blocked awaiting https://osgeo-org.atlassian.net/browse/GEOT-5860

**Pull request closed without merging**